.config/nvim/ftplugin/java.lua

```diff
@@ -0,0 +1,83 @@
+-- If you started neovim within `~/dev/xy/project-1` this would resolve to `project-1`
```

Comment:
Remove all the comments here - busy file, and these comments are available on the repo. Maybe drop a link to the relevant repos :)